src/main/java/edu/ucla/library/iiif/fester/verticles/ManifestVerticle.java

```diff
@@ -103,7 +104,8 @@ public void start(final Promise<Void> aPromise) {
                 final String action = message.headers().get(Constants.ACTION);
                 final Path filePath = Paths.get(body.getString(Constants.CSV_FILE_PATH));
                 final String iiifVersion = body.getString(Constants.IIIF_API_VERSION);
-                final CsvParser csvParser = new CsvParser().parse(filePath, iiifVersion);
+                final CsvParser csvParser = new CsvParser().setAVUrlString(config().getString(Config.AV_URL_STRING)) //, DEFAULT_AV_STRING))
```

Comment:
Since this line is calling the constructor, why not pass the configuration directly to it, rather than creating a new public setter method? If the programmer forgets the call to setAVUrlString, the code will throw a NullPointerException from within the `parse` method, which could potentially be confusing (admittedly not much of a practical concern now, since this is the only place its used, but I think in general we should try not to write code that allows errors like that)